### DISCUSSION TITLE
View Function Logs gives this jinja2

### DISCUSSION THREAD

**Discussion by BunyaminEfe** [vercel/community#1128] [Help] [answered]
![image](https://user-images.githubusercontent.com/30906513/206436901-a08eff88-d30d-4d76-b6b2-91a7485fe5e9.png)
[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': cannot import name 'escape' from 'jinja2' (/var/task/jinja2/__init__.py)
Traceback (most recent call last): 

What is the solution for the error?

**Comment by amyegan** [comment, marked as answer]
Hi @BunyaminEfe! I think this error is related to the removal of `escape` module support in flask/jinja. To fix it, you could [update the flask version](https://stackoverflow.com/a/71735103) or [limit the jinja version](https://stackoverflow.com/a/71721006).

**Reply by BunyaminEfe** [reply]
@amyegan wow i never thought of it that way i will try it and get back to you :)

**Comment by BunyaminEfe** [comment]
@amyegan thanks i solved the problem
